WordPressKit/WPCountry.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-@objc public class State: NSObject, Codable {
+@objc public class WPCountry: NSObject, Codable {
```

Comment:
lol at `git` considering that `State.swift` got renamed into `WPCountry.swift` and class changed to `State` -> `WPCountry`… and vice-versa for `Country` -> `WPState` on the other file… thanks `git` 🤦  (aka why do it simple when you can make it complicated 😆 )